.github/sync-repo-settings.yaml

```diff
@@ -0,0 +1,48 @@
+# Whether or not rebase-merging is enabled on this repository.
```

Comment:
is there any new logic in here? Or just migrating repo settings to this repo?

Reply:
It should match the same settings currently on the repository.

Reply:
Oh although, maybe we api-bigtable instead of actools as admins (or push?)